### DISCUSSION TITLE
Question about strapi-plugin-comments

### DISCUSSION THREAD

**Discussion by dnikomon** [VirtusLab-Open-Source/strapi-plugin-comments#147] [Q&A]
Do I have to create my own user interface and this solution provides the api and the models?

I have been looking where I can post a comment in strapi interface and didn't quite understand what I had done wrong during installation?

TL;DR To actually post comments, I have to implement own plugin?

**Comment by cyp3rius** [comment]
This is a plugin which expose API (REST + GQL) for apps based on Strapi which is Headless CMS => no visualisation, just data. If you've got app filled by data from Strapi, you must create part for comments as well (list, form etc).
